Assets/MixedRealityToolkit.Tests/TestUtilities.cs

```diff
@@ -54,16 +106,19 @@ public static void InitializePlayspace()
             });
         }
 
-        public static void InitializeMixedRealityToolkitScene(bool useDefaultProfile = false)
+        public static void InitializeMixedRealityToolkitScene(bool useDefaultProfile = false, int numScenesToCreate = 1)
         {
             // Setup
-            CleanupScene();
-
-            MixedRealityToolkit mixedRealityToolkit = new GameObject("MixedRealityToolkit").AddComponent<MixedRealityToolkit>();
-            MixedRealityToolkit.SetActiveInstance(mixedRealityToolkit);
+            CreateScenes(numScenesToCreate);
+            InitializeMixedRealityToolkit(useDefaultProfile);
+        }
 
+        public static void InitializeMixedRealityToolkit(bool useDefaultProfile = false)
+        {
             if (!MixedRealityToolkit.IsInitialized)
```

Comment:
It's suspicious that MRTK can be initialized once, but then that multiple scenes with MRTK can be loaded. What is the determininstic outcome of this method? If one scene with MRTK exists, then MRTK for that scene will be loaded (hopefully, actualy may not be true).

If there are several scenes with MRTK, does this mean that the version of MRTK that will be loaded / initialized is the MRTK from the very first scene?

Reply:
>If there are several scenes with MRTK, does this mean that the version of MRTK that will be loaded / initialized is the MRTK from the very first scene?

Yes, the first MRTK instance initialized will remain the active instance until you manually switch it with `MixedRealityToolkit.SetActiveInstance.` All other instances will be inactive.